metaplex/js/test/setup/build.ts

```diff
@@ -0,0 +1,73 @@
+import stream from 'stream';
+import path from 'path';
+import { promisify } from 'util';
+import { execSync } from 'child_process';
+import { renameSync, rmSync, createWriteStream, mkdirSync } from 'fs';
+import { tmpTestDir } from '../utils';
+import axios from 'axios';
+import StreamZip from 'node-stream-zip';
+
+const rustGithubRepository = 'metaplex-program-library';
+const repositoryDir = `${rustGithubRepository}-master`;
+const rustProgramsRepository = `https://github.com/metaplex-foundation/${rustGithubRepository}/archive/refs/heads/master.zip`;
+
+const isLocal = process.env.LOCAL_MPL === '1';
+
+function outputAndExitError(error: Error): void {
+  console.error(`${error.name}: ${error.message}`);
+  process.exit(1);
+}
+
+async function build() {
+  const programs: string[] = [
+    'auction/program',
+    'token-metadata/program',
+    'token-vault/program',
+    'metaplex/program',
+  ];
+  rmSync(tmpTestDir, { recursive: true, force: true });
+  mkdirSync(tmpTestDir);
+
+  async function downloadFile(path: string, destinationPath: string): Promise<void> {
+    const done = promisify(stream.finished);
+    const writer = createWriteStream(destinationPath);
+    try {
+      const response = await axios({ method: 'get', url: path, responseType: 'stream' });
+      response.data.pipe(writer);
+      await done(writer);
+    } catch (error) {
+      outputAndExitError(error);
+    }
+  }
+
+  if (!isLocal) {
```

Comment:
`isLocal` was so the JS SDK repo could either download MPL code from github or use a local copy. We don't need `isLocal` here since we already have all of the MPL code. Make the `isLocal` logic the default behavior (build the local code) and remove the non-isLocal logic.

Reply:
Done